c/driver/flightsql/CMakeLists.txt

```diff
@@ -63,6 +63,18 @@ if(ADBC_BUILD_TESTS)
                 adbc_driver_common
                 adbc_validation
                 ${TEST_LINK_LIBS})
+
+  if(ADBC_TEST_LINKAGE STREQUAL "shared")
+    # The go build is not copying the DLL to the test location, causing the test to fail on Windows.
```

Comment:
Hmm, how is this handled for the C/C++ builds? I wouldn't think CMake copies the DLL over...

Reply:
It seems by default cmake copies the dll when it is referenced as a library by another cmake project (the test one in this case). However, because the flightsql and snowflake ones are generated manually via GoUtils, they are not getting copied to the test bin folder.